ci/infra/testrunner/tests/test_node_reboot.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import json
+import time
+
+import pytest
+from timeout_decorator import timeout
+
+logger = logging.getLogger("testrunner")
+
+
+@timeout(120)
+def wait_for_node_reboot(node_type, node_number, platform):
```

Comment:
I think the logic for rebooting a node (and waiting for it to come back alive) could be moved to the platform to make it reusable.